Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM golang:1.17-alpine3.14
+FROM golang:1.17-alpine3.16
+
+# Update
+RUN apk update && apk upgrade
```

Comment:
is this req with --no-cache specified for installs?

Reply:
great point. I'll add this in.